data/brands/shop/doityourself.json

```
@@ -246,6 +246,16 @@
         "shop": "doityourself"
       }
     },
+    {
+      "displayName": "Bricopro",
+      "locationSet": {"include": ["fr"]},
```

```suggestion
      "locationSet": {"include": ["fx"]},
```
Their website only shows locations in Metropolitan France (`fr` includes all French territories)